homeassistant/components/withings/sensor.py

```diff
@@ -14,6 +14,11 @@
     WithingsDataManager,
 )
 
+# There's only 3 calls (per profile) made to the withings api every 5
+# minutes (see throttle values). This component wouldn't benefit
+# much from parallel updates.
+PARALLEL_UPDATES = 0
```

Comment:
Set it to 1 if we only want 1 update at a time.

Setting it to 0 will mean limitless parallel updates.